okhttp/src/main/java/com/squareup/okhttp/ConnectionSpec.java

```diff
@@ -242,7 +242,7 @@ public Builder cipherSuites(CipherSuite... cipherSuites) {
       return cipherSuites(strings);
     }
 
-    Builder cipherSuites(String[] cipherSuites) {
+    public Builder cipherSuites(String[] cipherSuites) {
```

Comment:
Well I only ask because your test case is written with a single string which made me assume it's what you meant.